railties/lib/rails/generators/rails/app/templates/test/test_helper.rb.tt

```diff
@@ -4,10 +4,10 @@ require "rails/test_help"
 
 class ActiveSupport::TestCase
   # Run tests in parallel with specified workers
-<% if defined?(JRUBY_VERSION) || Gem.win_platform? -%>
-  parallelize(workers: :number_of_processors, with: :threads)
-<%- else -%>
+<% if Process.respond_to?(:fork) -%>
```

Comment:
We need to keep the Windows platform check here, it's not only about fork support (see https://github.com/rails/rails/pull/34410).

```suggestion
<% if Process.respond_to?(:fork) && !Gem.win_platform? -%>
```

Reply:
@eugeneius Okay, I've updated to add ` && !Gem.win_platform?`. Thanks.